### DISCUSSION TITLE
Quick-look AR is not working on Chrome, Firefox and Edge on IOS

### DISCUSSION THREAD

**Discussion by matios13** [google/model-viewer#3424] [Q&A] [answered]
### Description
When you try to enter AR on any webbroser other than safari on IOS you will see usdz file as text rather than AR

#### Live Demo
https://modelviewer.dev/examples/augmentedreality/#ar
![image_123986672](https://user-images.githubusercontent.com/8004598/166905488-c66bf623-8786-43f2-9a86-8d45e807b872.JPG)

### Version

- model-viewer: v1.11.1

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [x] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by mimeeq-neal** [comment, marked as answer]
Hi @matios13 , Solution can be found here https://github.com/google/model-viewer/issues/3292
